src/crimson/osd/main.cc

```diff
@@ -19,13 +19,13 @@
 
 #include "auth/KeyRing.h"
```

Comment:
Might be worth to turn the PR's opening message into the commit's description.

Reply:
done